src/bindings/pygaia/scripts/classification/train_model.py

```diff
@@ -26,8 +26,10 @@
 import generate_classification_project
```

Comment:
it would be nice to have a README in this folder which gives an overview of what the scripts are - what order to run them in for the main classification, and what the additional tools do